### PR TITLE
readme: add exps to quickstart

### PR DESCRIPTION
**Pull request by dberenbaum**
Added some exp commands to the quickstart

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -38,25 +38,29 @@ Quick start
...
-+-----------------------------------+----------------------------------------------------------------------------+
+
++-----------------------------------+----------------------------------------------------------------------------------+
+| Task                              | Terminal                                                                         |
++===================================+==================================================================================+
+| Track data                        | | ``$ git add train.py params.yaml``                                             |
+|                                   | | ``$ dvc add images.zip``                                                       |
++-----------------------------------+----------------------------------------------------------------------------------+
+| Connect code and data             | | ``$ dvc stage add -n prepare -d images.zip -o images/ unzip -q images.zip``    |
+|                                   | | ``$ dvc stage add -n train -d images/ -d train.py -o model.p python train.py`` |
```
Comment (dberenbaum) [inline README.rst:49]: Do we need a `dvc exp run` or `dvc repro` call here?

File: README.rst [hunk 2]
```diff
@@ -38,25 +38,29 @@ Quick start
...
-+-----------------------------------+----------------------------------------------------------------------------+
-| Share data and ML models          | | ``$ dvc remote add myremote -d s3://mybucket/image_cnn``                 |
-|                                   | | ``$ dvc push``                                                           |
-+-----------------------------------+----------------------------------------------------------------------------+
+
++-----------------------------------+----------------------------------------------------------------------------------+
+| Task                              | Terminal                                                                         |
++===================================+==================================================================================+
+| Track data                        | | ``$ git add train.py params.yaml``                                             |
+|                                   | | ``$ dvc add images.zip``                                                       |
```
Comment (shcheklein) [inline README.rst:46]: remove `.zip` from images I think. make it `dvc add images/`

File: README.rst (hunk 1 above)
Comment (shcheklein) [reply README.rst:49]: makes sense, yep

Comment (shcheklein) [reply README.rst:49]: actually it comes at the next step?

File: README.rst [hunk 3]
```diff
@@ -38,25 +38,29 @@ Quick start
...
++-----------------------------------+----------------------------------------------------------------------------------+
+| Compare and select experiments    | | ``$ dvc exp show``                                                             |
+|                                   | | ``$ dvc exp apply exp-code-change``                                            |
++-----------------------------------+----------------------------------------------------------------------------------+
+| Share code                        | | ``$ git add .``                                                                |
+|                                   | | ``$ git commit -m 'The baseline model'``                                       |
+|                                   | | ``$ git push``                                                                 |
++-----------------------------------+----------------------------------------------------------------------------------+
+| Share data and ML models          | | ``$ dvc remote add myremote -d s3://mybucket/image_cnn``                       |
```
[the remark this answers is not in the document]
Comment (dberenbaum) [reply README.rst:61]: It gets really long, but we should probably add metrics if we are showing experiments. Another option would be `vi dvc.yaml` and maybe linking to an example of what that should look like.

File: README.rst (hunk 2 above)
Comment (dberenbaum) [reply README.rst:46]: I thought the point was to show a multi-step pipeline in the next step. Do you think we should get rid of that, or make up some new transformation as the first step?

Comment (shcheklein) [reply README.rst:46]: I see, yes, let's keep ... this is not good though by itself (bad pattern for DVC to use archives)

Comment (dberenbaum) [reply README.rst]: Made a new transformation

File: README.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (dberenbaum) [reply README.rst]: Added metrics

**Review by daavoo — APPROVED** [review]

**Comment by shcheklein** [comment]
@dberenbaum looks good to me 😎 let's 🚢 it

**Review by shcheklein — APPROVED** [review]

**Comment by dberenbaum** [comment]
🙏 @daavoo Could you merge please?

**Pull request merged**